plugins/main/bridge/bridge.go

```diff
@@ -395,20 +405,45 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return fmt.Errorf("cannot set hairpin mode and promiscous mode at the same time.")
 	}
 
-	br, brInterface, err := setupBridge(n)
+	bridgeNS, err := ns.GetCurrentNS()
```

Comment:
why not just load bridge namespace here if non empty?

Reply:
I wanted to have the same logic to follow up everywhere. So I need to populate bridgeNS to the current host network namespace, so I can work with it the same way.

Is my assumption wrong here?